docs/3.0.0-alpha.x/getting-started/quick-start-tutorial.md

```diff
@@ -141,21 +135,13 @@ After your admin user is registered, you will see the Strapi admin panel:
 
 ![Strapi Admin Panel](../assets/quick-start-detailed/AfterRegistrationScreenAdminPanel.png 'Strapi Admin Panel')
 
----
-
-Every user is found by clicking in the left menu: **CONTENT TYPES** --> **Users**.
-
-From here you can add and edit users.
-
-![Strapi Users Page](../assets/quick-start-detailed/AdminUserInUsers.png 'Strapi Users Page')
-
 ## 4. Create a Content Type
 
 At this point, your project is empty. You need to create a **Content Type**.
 
-We can define a **Content Type** as a collection of entered data represented by fields.  For example, a **Content Type** called `restaurant` may be intended to display information regarding restaurants. A `restaurant` has a `name`, a main `image`, a `description`, maybe even a link to the `restaurant menu`.
+We can define a **Content Type** as a collection of entered data represented by fields. For example, a **Content Type** called `restaurant` may be intended to display information regarding restaurants. A `restaurant` has a `name`, a main `image`, a `description`, maybe even a link to the `restaurant menu`.
 
-Another example, is a **Content Type** called `menu` which is made up of `menu items`.  These have a `name`, `description of the menu item`, an `image` and perhaps even a `list of ingredients`.
+Another example, is a **Content Type** called `menu` which is made up of `menu items`. These have a `name`, `description of the menu item`, an `image` and perhaps even a `list of ingredients`.
```

Comment:
i think `cover` would be more accurate than `image`

Reply:
For the reason discussed in the other repo. This would create a change that affects to many pieces and not really necessary.  Though for the future a good idea.